Formula/k/kubectl-cnpg.rb

```diff
@@ -0,0 +1,32 @@
+class KubectlCnpg < Formula
+  desc "CloudNativePG plugin for kubectl"
+  homepage "https://cloudnative-pg.io/"
+  url "https://github.com/cloudnative-pg/cloudnative-pg/archive/refs/tags/v1.23.1.tar.gz"
+  sha256 "c13373baf2a0d177d82f85be66b963ff6d55853aaafc7cbbe9afa904d75b8695"
+  license "Apache-2.0"
+  head "https://github.com/cloudnative-pg/cloudnative-pg.git", branch: "main"
+
+  depends_on "go" => :build
+
+  def install
+    ldflags = %W[
+      -s -w
+      -X github.com/cloudnative-pg/cloudnative-pg/pkg/versions.buildVersion=#{version}
+      -X github.com/cloudnative-pg/cloudnative-pg/pkg/versions.buildCommit=#{tap.user}
+      -X github.com/cloudnative-pg/cloudnative-pg/pkg/versions.buildDate=#{time.iso8601}
+    ]
+    system "go", "build", *std_go_args(ldflags:, output: "#{bin}/kubectl-cnpg"), "./cmd/kubectl-cnpg"
```

Comment:
```suggestion
    system "go", "build", *std_go_args(ldflags:), "./cmd/kubectl-cnpg"
```
I think the `output` you specify is already the default.

Reply:
Yup you're correct, fixed.